geometry_msgs/Point beta0
geometry_msgs/Point beta1
geometry_msgs/Point beta2
uint32 len
